### PR TITLE
Enable Agenda 🗓️

### PR DESCRIPTION
**Pull request by agustints**

### CODE REVIEW

File: src/app/(web)/(content)/(meetups)/(2024)/la-meetup/services/getAgenda.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
+import { getPayloadHMR } from "@payloadcms/next/utilities";
 import { type ElementNode } from "@payloadcms/richtext-slate";
 
-import { PAYLOAD_API_URL } from "app/lib/constants";
+import configPromise from "../../../../../../../admin/payload.config";
```
Comment (jmoliugp) [inline src/app/(web)/(content)/(meetups)/(2024)/la-meetup/services/getAgenda.ts:4]: Absolute paths might be helpful in a near future.

File: src/components/Meetups/2024/Agenda/index.tsx [hunk 2]
```diff
@@ -57,7 +59,7 @@ export default function Agenda({ title, subtitle, lastUpdate, agenda }: AgendaPr
                     <div className="flex w-full flex-row items-center justify-between gap-3 text-sm md:text-lg">
                       <div className="flex flex-row items-center justify-between gap-3">
                         <span className="text-yellow-400 md:mr-3">
-                          {format(startTime, "HH:mm")} - {format(endTime, "HH:mm")}
+                          {format(new UTCDate(startTime), "HH:mm")} - {format(new UTCDate(endTime), "HH:mm")}
```
Comment (jmoliugp) [inline src/components/Meetups/2024/Agenda/index.tsx:62]: Is the `UTCDate` really required?

**Review by jmoliugp — APPROVED** [review]
LGTM.

Left some minor comments.

File: src/components/Meetups/2024/Agenda/index.tsx (hunk 2 above)
Comment (agustints) [reply src/components/Meetups/2024/Agenda/index.tsx]: The server was originally returning UTC dates, but I updated the configuration, so it's no longer necessary!

**Pull request merged**